modules/xdcr-reference/pages/xdcr-filtering-data-type-conversion.adoc

```diff
@@ -154,171 +154,171 @@ Notes on comparison-procedures are also provided.
 | Collation comparison is performed for all except `Regex`.
 
 | 15
 | `<Any-Except-Pcre>`
 | `Pcre`
-| image:xdcr-reference:cross-mark-red-sm-wide.png[] Invalid.
+| image:cross-mark-red-sm-wide.png[] Invalid.
 | Collation comparison is performed for all except `Pcre`.
 
 | 16
 | `<Any-Except-Null>`
 | `Null`
-| image:xdcr-reference:checkmark-green-sm.png[] Valid.
+| image:checkmark-green-sm.png[] Valid.
 `<Any>` is converted a `non-Null` value, for comparison with `Null`.
 | The comparison `non-Null > Null` returns `true`.
 
 | 17
 | `Int`
 | `Boolean`
-| image:xdcr-reference:checkmark-green-sm.png[] Valid for all `Int` values.
+| image:checkmark-green-sm.png[] Valid for all `Int` values.
 The `Int` value `0` is converted to the `Boolean` value `false`; all other `Int` values are converted to the `Boolean` value `true`.
 | The boolean-comparison `true > false` returns `true`.
 
 | 18
 | `Uint`
 | `Boolean`
-| image:xdcr-reference:checkmark-green-sm.png[] Valid for all `Uint` values.
+| image:checkmark-green-sm.png[] Valid for all `Uint` values.
 The `Uint` value `0` is converted to the `Boolean` value `false`; all other `Uint` values are converted to the `Boolean` value `true`.
 | The boolean-comparison `true > false` returns `true`.
 
 | 19
 | `Float`
 | `Boolean`
-| image:xdcr-reference:checkmark-green-sm.png[] Valid for all `Float` values.
+| image:checkmark-green-sm.png[] Valid for all `Float` values.
 The `Float` value `0.0` is converted to the `Boolean` value `false`; all other `Float` values are converted to the `Boolean` value `true`.
 | The boolean-comparison `true > false` returns `true`.
 
 | 20
 | `String`
 | `Boolean`
-| image:xdcr-reference:checkmark-orange-sm.png[] Valid if `String` is case-insensitive `"true"` or `"false"`; in which case `String` is converted to its `Boolean` equivalent.
+| image:checkmark-orange-sm.png[] Valid if `String` is case-insensitive `"true"` or `"false"`; in which case `String` is converted to its `Boolean` equivalent.
 | The boolean-comparison `true > false` returns `true`.
 
 | 21
 | `<Any-Except-Array>`
 | `Array`
-| image:xdcr-reference:cross-mark-red-sm-wide.png[] Invalid.
+| image:cross-mark-red-sm-wide.png[] Invalid.
 | Collation comparison is performed.
 
 | 22
 | `<Any-Except-Object>`
 | `Object`
-| image:xdcr-reference:cross-mark-red-sm-wide.png[] Invalid.
+| image:cross-mark-red-sm-wide.png[] Invalid.
 | Collation comparison is performed. is used.
 
 | 23
 | `<Any-Except-Time-and-String`
 | `Time`
-| image:xdcr-reference:cross-mark-red-sm-wide.png[] Invalid.
+| image:cross-mark-red-sm-wide.png[] Invalid.
 | Collation comparison is performed.
 
 | 24
 | `String`
 | `Time`
-| image:xdcr-reference:checkmark-orange-sm.png[] Valid if `String` can be parsed as a parameter to the `DATE` function.
+| image:checkmark-orange-sm.png[] Valid if `String` can be parsed as a parameter to the `DATE` function.
 | Standard comparision is performed if valid; otherwise, collation comparison is performed.
```

Comment:
Typo `comparision` → `comparison`

Please find and replace — this typo occurs in 5 locations in this document. I haven't marked them all.